neuromation/cli/main.py

```diff
@@ -178,6 +178,16 @@ def print_options(
     is_flag=True,
     help="Trace sent HTTP requests and received replies to stderr.",
 )
+@click.option(
+    "--hide-token/--no-hide-token",
+    is_flag=True,
+    default=None,
+    help=(
+        "Prevent user's token sent in HTTP headers from being "
+        "printed out to stderr during HTTP tracing. Can be used only "
+        "together with option '--trace'  [default: True]"
```

Comment:
`default: True` is meaningless for flags; I have no idea what option matches to `True` or `False` on the reading.

Reply:
1. it is much more clear than `--hide-token` (as True) and `--show-token` (as False).
2. if still it is meaningless, let's then fix all other similar places:

https://github.com/neuromation/platform-client-python/blob/72416311c23a0eb6c0e99053c53268d5e1efc977/neuromation/cli/job.py#L151-L158

https://github.com/neuromation/platform-client-python/blob/72416311c23a0eb6c0e99053c53268d5e1efc977/neuromation/cli/job.py#L162-L167

https://github.com/neuromation/platform-client-python/blob/72416311c23a0eb6c0e99053c53268d5e1efc977/neuromation/cli/job.py#L168-L174

https://github.com/neuromation/platform-client-python/blob/72416311c23a0eb6c0e99053c53268d5e1efc977/neuromation/cli/job.py#L222-L227

https://github.com/neuromation/platform-client-python/blob/72416311c23a0eb6c0e99053c53268d5e1efc977/neuromation/cli/job.py#L228-L233

https://github.com/neuromation/platform-client-python/blob/72416311c23a0eb6c0e99053c53268d5e1efc977/neuromation/cli/job.py#L315-L321

https://github.com/neuromation/platform-client-python/blob/72416311c23a0eb6c0e99053c53268d5e1efc977/neuromation/cli/storage.py#L65-L71

https://github.com/neuromation/platform-client-python/blob/72416311c23a0eb6c0e99053c53268d5e1efc977/neuromation/cli/storage.py#L222-L228

If not, let's move on to do more important things.

Reply:
Honestly, I wanted to fix the places mentioned by you eventually.
"Show progress, on by default" from the last snippet is a good text IMHO.

The fix of existing options worth another PR, adding a new option with text that should be fixed from the very beginning is suboptimal.

Reply:
fixed message to `... On by default.`